src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -558,10 +562,14 @@ int AbortEnergyPlus(EnergyPlusData &state)
     const std::string Elapsed = state.dataSysVars->runtimeTimer.formatAsHourMinSecs();
 
     state.dataResultsFramework->resultsFramework->SimulationInformation.setRunTime(Elapsed);
+    state.dataResultsFramework->resultsFramework->SimulationInformation.setNumErrorsInputs(NumWarningsDuringInputs, NumSevereDuringInputs);
     state.dataResultsFramework->resultsFramework->SimulationInformation.setNumErrorsWarmup(NumWarningsDuringWarmup, NumSevereDuringWarmup);
     state.dataResultsFramework->resultsFramework->SimulationInformation.setNumErrorsSizing(NumWarningsDuringSizing, NumSevereDuringSizing);
     state.dataResultsFramework->resultsFramework->SimulationInformation.setNumErrorsSummary(NumWarnings, NumSevere);
 
+    ShowMessage(
+        state,
+        format("EnergyPlus Inputs Error Summary. During Inputs: {} Warning; {} Severe Errors.", NumWarningsDuringInputs, NumSevereDuringInputs));
```

Comment:
I could not get the number of warning/severe errors to match what was reported previously so went ahead and added a summary for errors during getInput.